styles/styles.css

```
@@ -717,11 +911,371 @@ body {
 }
 
 .main-footer {
+  display: flex;
   margin-bottom: 40px;
   padding: 40px 0;
 }
 
+.button-footer {
+  margin-left: auto;
+}
+
 .htmlacademy-link {
   text-decoration: none;
   color: #2d3440;
+}
+
+.navigation-select {
+  list-style-type: none;
+}
+
+.navigation-icon {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  margin: auto;
+  fill: #dbdbdb;
+}
+
+.navigation-link-login {
+  padding-left: 38px;
+  border-radius: 30px;
+}
+
+.navigation-link-login .navigation-icon {
+  right: auto;
+  left: 16px;
+  margin: auto 0;
+}
+
+.advantages-item {
+  display: flex;
+  gap: 16px;
+}
+
+.advantages-item::before {
+  content: '';
+  display: block;
+  flex-shrink: 0;
+  width: 48px;
+  height: 48px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.advantages-item-leaf::before {
+  background-image: url('../images/leaf.svg');
+}
+
+.advantages-item-ice-cream::before {
+  background-image: url('../images/ice-cream-mini.svg');
+}
+
+.advantages-item-cow::before {
+  background-image: url('../images/cow.svg');
+}
+
+.advantages-item-thermometre::before {
+  background-image: url('../images/thermometre.svg');
+}
+
+.breadcrumbs-item::after {
+  content: '';
+  width: 12px;
+  height: 12px;
+  margin: auto;
+  background-image: url('../images/double-arrow.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.breadcrumbs-item:last-child::after {
+  content: none;
+}
```

1. Я бы положил рядом со стилями для `breadcrumbs-item` на 816 строке которые
2. По сути нам тут надо всех, кроме последнего - напрашивается селектор `:not` и тогда не надо будет напрямую удалять у последнего

```suggestion
.breadcrumbs-item:not(:last-child)::after {
  content: '';
  width: 12px;
  height: 12px;
  margin: auto;
  background-image: url('../images/double-arrow.svg');
  background-repeat: no-repeat;
  background-position: center;
}
```